src/components/Banner/BaseAnnouncementBanner/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Trans } from '@lingui/macro'
+import { InterfaceElementName } from '@uniswap/analytics-events'
+import { ReactComponent as AppleLogo } from 'assets/svg/apple_logo.svg'
+import baseLogoUrl from 'assets/svg/base_background_icon.svg'
+import { useScreenSize } from 'hooks/useScreenSize'
+import { useLocation } from 'react-router-dom'
+import { useHideUniswapWalletBanner } from 'state/user/hooks'
+import { ThemedText } from 'theme'
+import { openDownloadApp, openWalletMicrosite } from 'utils/openDownloadApp'
+import { isIOS, isMobileSafari } from 'utils/userAgent'
+
+import { BannerButton, BaseBackgroundImage, ButtonRow, PopupContainer, StyledXButton } from './styled'
+
+export default function UniswapWalletBanner() {
+  const [hideUniswapWalletBanner, toggleHideUniswapWalletBanner] = useHideUniswapWalletBanner()
```

Comment:
Do we want to reset the state for this to pump Base launch? For example I was only able to see this banner in incognito